variables.tf

```diff
@@ -846,18 +850,6 @@ variable "docker_machine_egress_rules" {
   }]
 }
 
-variable "subnet_id_runners" {
-  description = "Deprecated! Use subnet_id instead. List of subnets used for hosting the gitlab-runners."
-  type        = string
-  default     = ""
-}
-
-variable "subnet_ids_gitlab_runner" {
-  description = "Deprecated! Use subnet_id instead. Subnet used for hosting the GitLab runner."
-  type        = list(string)
-  default     = []
-}
-
```

Comment:
issue: In general a good idea to remove theses outdated variables. But this creates extra work for all users to update their module call. With the next major release we will restructure and clean the inputs. Please add them again.